src/components/Restoration/CorrectNameRequest.vue

```diff
@@ -63,12 +61,23 @@
 <script lang="ts">
 import Vue from 'vue'
 import { Component, Prop, Watch, Emit } from 'vue-property-decorator'
-import { Action, Getter } from 'vuex-class'
-import { ConfirmDialog as ConfirmDialogShared } from '@bcrs-shared-components/confirm-dialog/'
-import { CommonMixin, NameRequestMixin } from '@/mixins/'
-import { ActionBindingIF, ConfirmDialogType, NameRequestIF, NrCorrectionIF, NrResponseIF } from '@/interfaces/'
-import { NameChangeOptions } from '@/enums/'
-import { CorpTypeCd, GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module/'
+import { ConfirmDialog as ConfirmDialogShared } from '@bcrs-shared-components/confirm-dialog'
+import { CommonMixin, NameRequestMixin } from '@/mixins' // *** TODO - verify this
+import { ConfirmDialogType, NameRequestIF, NrResponseIF } from '@bcrs-shared-components/interfaces'
+import { NameChangeOptions } from '@bcrs-shared-components/enums'
+import { CorpTypeCd, GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module'
+
+interface NrCorrectionIF {
+  legalType: CorpTypeCd,
+  nrNumber: string,
+  legalName: string,
+  expiry: string,
+  requestType: string,
+  status: NameRequestStates,
+  applicant: NameRequestApplicantIF | object
+}
```

Comment:
This may go away if I can reuse one of the other two (!) NR interfaces... Otherwise this is used here only.